### ISSUE TITLE
Can it support multiple symbol portfolio backtesting?

### ISSUE THREAD

**Issue by smartian1** [diogomatoschaves/stratestic#16]
Or is there a plan to support it in the future?

**Comment by diogomatoschaves** [comment]
Yes — portfolio backtesting across multiple symbols shipped in **2.1.0** and was refined in **2.2.0** (now on PyPI).

Pass a `(symbol, field)` MultiIndex panel (`stratestic.utils.panel.build_panel({...})`) to either backtester together with a multi-symbol strategy. The quickest path is `BroadcastStrategy`, which applies one strategy across every symbol:

```python
from stratestic.backtesting import VectorizedBacktester
from stratestic.strategies import MovingAverageCrossover
from stratestic.strategies.multi import BroadcastStrategy
from stratestic.utils.panel import build_panel

panel = build_panel({"BTCUSDT": btc_df, "ETHUSDT": eth_df})
bt = VectorizedBacktester(BroadcastStrategy(MovingAverageCrossover(20, 50), data=panel))
bt.run()
```

Equal-weight by default, with optional strategy-provided weights, shared capital, and per-symbol isolated margin. The portfolio equity plot now also shows each symbol's contribution. See the README's "Multi-symbol backtesting" section. Closing as done — `pip install -U stratestic`.
